src/index.js

```diff
@@ -1,197 +1,67 @@
-    PanelHead,
-    PanelActions,
-    PanelFilters,
-    PanelContent,
-    PanelFooter
-} from '../src/Panel/Panel';
-import { Popover } from '../src/Popover/Popover';
-import { SearchInput } from '../src/SearchInput/SearchInput';
-import { SideNav, SideNavList, SideNavGroup } from '../src/SideNavigation/SideNavigation';
-import { Table } from '../src/Table/Table';
-import { Tabs, TabComponent } from '../src/Tabs/Tabs';
-import { Token } from './Token/Token';
-import {
-    Tile,
-    TileContent,
-    TileMedia,
-    TileActions,
-    ProductTile,
-    ProductTileMedia,
-    TileGrid,
-    ProductTileContent
-} from '../src/Tile/Tile';
-import { Toggle } from '../src/Toggle/Toggle';
-import { Tree } from '../src/Tree/Tree';
-import { Time } from '../src/Time/Time';
-import { TimePicker } from '../src/TimePicker/TimePicker';
-import { Shellbar } from '../src/Shellbar/Shellbar';
-import { DocsTile, DocsText } from '../src/documentation/DocsTile/DocsTile';
-import { Separator } from '../src/documentation/Separator/Separator';
-import { Header } from '../src/documentation/Header/Header';
-import { Description } from '../src/documentation/Description/Description';
-import { Import } from '../src/documentation/Import/Import';
-import { Properties } from '../src/documentation/Properties/Properties';
-import { Playground } from '../src/documentation/Playground/Playground';
-
-export {
-    DocsTile,
-    DocsText,
-    Separator,
-    Header,
-    Description,
-    Import,
-    Properties,
-    Playground,
-    ActionBar,
-    ActionBarBack,
-    ActionBarHeader,
-    ActionBarActions,
-    Alert,
-    Badge,
-    ComboboxInput,
-    Label,
-    Status,
-    Counter,
-    Breadcrumb,
-    BreadcrumbItem,
-    Button,
-    ButtonGroup,
-    Calendar,
-    DatePicker,
-    Dropdown,
-    FormSet,
-    FormItem,
-    FormLabel,
-    FormInput,
-    FormRadio,
-    FormTextarea,
-    FormMessage,
-    FormSelect,
-    FormFieldset,
-    FormLegend,
-    Icon,
-    Identifier,
-    Image,
-    InlineHelp,
-    InputGroup,
-    FormGroup,
-    ListGroup,
-    ListGroupItem,
-    ListGroupItemActions,
-    ListGroupItemCheckbox,
-    LocalizationEditor,
-    MegaMenu,
-    MegaMenuList,
-    MegaMenuGroup,
-    Menu,
-    MenuList,
-    MenuItem,
-    MenuGroup,
-    Modal,
-    MultiInput,
-    Popover,
-    Pagination,
-    Panel,
-    PanelGrid,
-    PanelBody,
-    PanelHeader,
-    PanelHead,
-    PanelActions,
-    PanelFilters,
-    PanelContent,
-    PanelFooter,
-    SearchInput,
-    SideNav,
-    SideNavList,
-    SideNavGroup,
-    Table,
-    Tabs,
-    TabComponent,
-    Token,
-    Tile,
-    TileContent,
-    TileMedia,
-    TileActions,
-    Time,
-    TimePicker,
-    ProductTile,
-    ProductTileMedia,
-    TileGrid,
-    ProductTileContent,
-    Toggle,
-    Tree,
-    Shellbar
-};
-
-ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+export { ActionBar } from '../src/ActionBar/ActionBar';
```

Comment:
I think all of these should be rewritten to _NOT_ include the `src` directory as I am still seeing the `src` directory in the path in the `dist` directory.

So:
```
export { ActionBar } from '../src/ActionBar/ActionBar';
```
should become:
```
export { ActionBar } from './ActionBar/ActionBar';
```